src/shared/components/palette/case-flag/components/update-flag/update-flag.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="form-group" [formGroup]="formGroup">
+  <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="100">
```

Comment:
@tarikdogan-hmcts - `data-maxlength` should be `200` (please check the ticket)

Reply:
done